website/docs/d/vpc_ipam_pools.html.markdown

```diff
@@ -0,0 +1,79 @@
+---
+subcategory: "VPC IPAM (IP Address Manager)"
+layout: "aws"
+page_title: "AWS: aws_vpc_ipam_pools"
+description: |-
+    Returns details about the first IPAM pool that matches search parameters provided.
```

Comment:
Description needs to be changed since it returns details about all IPAM pools that match the filter

```suggestion
    Returns details about IPAM pools that match the search parameters provided.

Reply:
good catch, thank you